daffodil-test-ibm1/src/test/resources/test-suite/tresys-contributed/BC.dfdl.xsd

```diff
@@ -47,7 +47,7 @@
         </sequence>
         <sequence dfdl:separator="," dfdl:terminator="%NL;">
           <element name="y" type="xsd:boolean" maxOccurs="unbounded"
-            dfdl:textBooleanTrueRep="{ 1*10 }" dfdl:textBooleanFalseRep="{'a b c'}" />
+            dfdl:textBooleanTrueRep="{ xsd:string(1*10) }" dfdl:textBooleanFalseRep="{'a b c'}" />
```

Comment:
Not sure it's worth investigating for this, but shouldn't DPath be able to implicitly coerce the number into a string? Or is this is a limitation with IBM's implementation that it does not do coercion?

Reply:
Will raise for clarification in DFDL Workgroup. Whether these coercion are implicit or explicit isn't clear in the DFDL spec for me. 

When to implicitly convert type is always a subtle thing in any language. It helps declutter things if simple implicit conversions are done. But if types are implicitly converted most of the time, then static type checking can't really find any errors for you, since a coercion can always be inserted. 

In this particular case, I think a static error is actually a better thing. Why would you write 1*10 for textBooleanTrueRep? If someone writes a numeric expression for something that's clearly supposed to be a string, probably they're just making a mistake.